### ISSUE TITLE
Bitfinex2 fee_to_precision raise KeyError

### ISSUE THREAD

**Issue by bobbytaxbit** [ccxt/ccxt#6614]
- OS: Ubuntu
- Programming Language version: Python 3.8
- CCXT version: 1.24.4
- Exchange:Bitfinex2
- Method: fee_to_precision via fetch_trades

Here's the output from my iPython shell with verbose = True

```
/usr/local/lib/python3.7/site-packages/ccxt/bitfinex2.py in fetch_my_trades(self, symbol, since, limit, params)
    873             method = 'privatePostAuthRTradesSymbolHist'
    874         response = getattr(self, method)(self.extend(request, params))
--> 875         return self.parse_trades(response, market, since, limit)
    876 
    877     def create_deposit_address(self, code, params={}):

/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py in parse_trades(self, trades, market, since, limit, params)
   1597     def parse_trades(self, trades, market=None, since=None, limit=None, params={}):
   1598         array = self.to_array(trades)
-> 1599         array = [self.extend(self.parse_trade(trade, market), params) for trade in array]
   1600         array = self.sort_by(array, 'timestamp')
   1601         symbol = market['symbol'] if market else None

/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py in <listcomp>(.0)
   1597     def parse_trades(self, trades, market=None, since=None, limit=None, params={}):
   1598         array = self.to_array(trades)
-> 1599         array = [self.extend(self.parse_trade(trade, market), params) for trade in array]
   1600         array = self.sort_by(array, 'timestamp')
   1601         symbol = market['symbol'] if market else None

/usr/local/lib/python3.7/site-packages/ccxt/bitfinex2.py in parse_trade(self, trade, market)
    566             if feeCost is not None:
    567                 fee = {
--> 568                     'cost': float(self.fee_to_precision(symbol, abs(feeCost))),
    569                     'currency': feeCurrency,
    570                 }

/usr/local/lib/python3.7/site-packages/ccxt/base/exchange.py in fee_to_precision(self, symbol, fee)
   1217 
   1218     def fee_to_precision(self, symbol, fee):
-> 1219         return self.decimal_to_precision(fee, ROUND, self.markets[symbol]['precision']['price'], self.precisionMode)
   1220 
   1221     def currency_to_precision(self, currency, fee):

KeyError: 'tBCHUSD'
```

It appears that an asset pair coming through as a fee is not in the markets data.

**Comment by kroitor** [comment]
Hi! Can you plz post your verbose output for the above request?

```Python
exchange.verbose = True
trades = exchange.fetch_my_trades ...
```

↑ Paste all the output here plz (without your keys, of course)

**Comment by bobbytaxbit** [comment]
Edited my original post with the output

**Comment by kroitor** [comment]
@bobbytaxbit what is the original symbol you're passing to `fetchMyTrades`?

**Comment by bobbytaxbit** [comment]
@kroitor we have a wrapper class that has these two methods

```
def get_trades(self) -> List[Dict[str, Any]]:
        if self.client.has['fetchMyTrades']:
            return self.trade_pagination_fn(self.client.fetch_my_trades)

def trade_pagination_fn(self, method: Callable) -> List[Dict[str, Any]]:
        end = int(time.time() * 1000)
        resp = method(since=1325376000000, params={'end': end}, limit=1000)  # Jan 1 2012, when Bitfinex started
        ret_val = resp
        while len(resp) == 1000:
            resp = method(since=1325376000000, params={'end': end}, limit=1000)
            ret_val += resp
            end = resp[-1]['timestamp'] - 1
        return ret_val
```

So it doesn't appear that we're passing in any symbol as we paginate.

**Comment by kroitor** [comment]
@bobbytaxbit it should be fixed in CCXT 1.24.7+, the new CI build will arrive in 15 minutes. Thx for reporting! It will be awesome if you could also post whether the new version works on your side or not.

**Comment by bobbytaxbit** [comment]
@kroitor Absolutely! Thanks for the prompt response!!!
